enhancements/storage/csi-vsphere-operator.md

```diff
@@ -152,6 +152,62 @@ If we detect an existing driver present in the cluster then cluster will be mark
 
 We will ensure that these steps are documented in 4.10 docs.
 
+#### Configure CSI driver topology
+
+A customer may configure a topology for the CSI driver by using following fields in `ClusterCSIDriver` object:
+
+```go
+// ClusterCSIDriverSpec is the desired behavior of CSI driver operator
+type ClusterCSIDriverSpec struct {
+    ...
+    ...
+    // `driverConfig` field can be used to optionally specify
+    // driver specific configuration.
+    // If this field is nil, platform chosen default configuration
+    // will be applied.
+    DriverConfig *CSIDriverConfigSpec `json:"driverConfig,omitempty"`
+}
+
+// CSIDriverConfigSpec defines configuration spec that can be
+// used to optionally configure a specific CSI Driver.
+// +union
+type CSIDriverConfigSpec struct {
+    // `driverName` indicates which of the driver we are
+    // installing
+    // +unionDiscriminator
+    // +required
+    DriverName CSIDriverName `json:"driverName"`
+
+    // `vsphere` field is used to configure vsphere CSI driver
+    // specific properties
+    // +optional
+    VSphere *VSphereCSIDriverConfigSpec `json:"vsphere"`
+}
+
+type VSphereCSIDriverConfigSpec struct {
+    // `topologyCategories` indicates categories with which
+    // vcenter resources as hostcluster or datacenter were tagged with.
+    // If unspecified CSI driver configuration will default to configuration
+    // specified in cluster configuration.
```

Comment:
I'm not really sure what this means, is this saying something that has been configured by the user or something that the CSI driver will create?

Where does the cluster configuration live? Is that an openshift thing?

Is this referring to vSphere tag categories?